plugins/pulp_docker/plugins/importers/sync.py

```diff
@@ -89,6 +141,34 @@ def generate_download_requests(self):
         for unit in self.step_get_local_blobs.units_to_download:
             yield self.index_repository.create_blob_download_request(unit.digest)
 
+    def v1_generate_download_requests(self):
```

Comment:
Can this live in the v1_sync module instead of here? It'd be nice to keep the v1/v2 stuff as separate as possible for the day when we remove v1.